docs/qupath.rst

```diff
@@ -57,11 +67,19 @@ Step-by-step
 
    |image1|
 
+#. Click ``Import``.
+
+#. The image thumbnail will appear in the left-hand pane list of the QuPath Project. Click on that thumbnail to open the image in full viewer in central pane of QuPath.
+
 #. Set image type to ``Brightfield H&E`` in the following dialog. Click ``OK``.
 
-#. Find a region with well-defined cells and nuclei in the image, zoom in.
+#. Find your ROIs from OMERO now in QuPath on that image.
+
+#. To draw new ROIs or annotations in QuPath, find a region with well-defined cells and nuclei in the image, zoom in.
+
+#. Draw an ``ROI Annotation`` which denotes the region in which the cells will be detected using the ``Wand`` tool |image2|. 
 
-#. Draw an ``ROI Annotation`` which denotes the region in which the cells will be detected using the ``Wand`` tool |image2|.
+#. Note that in case the ``ROI Annotation`` is encompassing a very large area, you might later get performance problems with the ``OME_XML_export.groovy`` script which exports the ``ROI Annotation`` in ome-xml format, because this script is attempting to export the ``ROI Annotation`` as a mask, see below.
```

Comment:
```suggestion
#. Note that if the ``ROI Annotation`` is encompassing a very large area, you might later get performance problems with the ``OME_XML_export.groovy`` script which exports the ``ROI Annotation`` in ome-xml format, because this script is attempting to export the ``ROI Annotation`` as a mask, see below.
```